website/html/forms.py

```diff
@@ -95,8 +106,9 @@ def clean_email(self):
 
 
 
-
-def gen_edit_user_form(field_list=None, *args, **kwargs):
+# Part of fix for Issue #152: change default field_list from None to '__all__'
+# for Django 1.8 compatibility
```

Comment:
56af63031b79319663e96c5eec3cb608b7080d90 removes the comment. (I think the comment should really be the commit message for that change.)